### DISCUSSION TITLE
Crafting categories

### DISCUSSION THREAD

**Discussion by CaitSith2** [ProfessorConfundus/PCs-Infinite-Ammo#4] [Ideas]
Given that you do a deep copy of things, do you think you could make sure the crafting category of ALL of the infinite ammo gets set to just crafting.  I noticed when I decided to try to craft an infinite flamethrower ammo that I couldn't handcraft it, despite its ingredients being just flamethrower ammo and stone, one of each.  It in fact had to be made in the same building as the original flamethrower ammo.

**Comment by ProfessorConfundus** [comment]
This is a good idea actually, thanks! I'll get to implementing that soon. I didn't think about changing the crafting category when making the mod initially. 
Thanks again for the idea!

**Comment by ProfessorConfundus** [comment]
I knew I should've tested more on non-sandbox worlds. I just assumed that'd be what most play it with, just after turning off god mode.

**Comment by ProfessorConfundus** [comment]
Just want to note, I'm really busy at the moment, so the update might take a while to be developed. Sorry for any inconveniences, @CaitSith2!

**Comment by ProfessorConfundus** [comment]
@CaitSith2 finally, the issue has been solved! I reworked the recipe generation system for another issue that was recently raised, and it just so happened to fix this too! Thank you again for reporting, and I'm so sorry for the long wait; I was really stumped on this seemingly simple problem! If you have any issues with this patch not working, please start an issue and I'll get right on it!
